test/functional/test_framework/script.py

```diff
@@ -14,8 +14,7 @@
 import sys
 bchr = chr
 bord = ord
-if sys.version > '3':
-    long = int
+if sys.version_info.major == 3:
```

Comment:
functional tests always run in Python 3. I think you can remove support for 2 entirely from this module.